modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/app-banner/index.css

```diff
@@ -1,52 +1,297 @@
 .banner {
-	background-color: #f7f7f8;
-	border-radius: 1rem;
-	box-shadow: 0 0.3125rem 0.9375rem 0 rgba(19, 20, 31, 0.12);
-	gap: 1.5rem;
-	height: 7.5rem;
+	background: var(--neutral-01, #f7f7f8);
+	border-radius: 10px;
+	box-shadow: 0px 5px 15px 0px rgba(19, 20, 31, 0.12);
+	padding: 12px;
 }
 
 .banner__content {
-	gap: 1.5rem;
+	flex-direction: row;
+	flex-flow: wrap;
 }
 
-.banner__content-app-logo {
-	height: 4.5rem;
-	width: 4.5rem;
+.banner__content__brand {
+	color: var(--neutral-08, #54555f);
+	display: none;
+	font-size: 16px;
+	font-style: normal;
+	font-weight: 400;
+	line-height: 24px;
+}
+.banner__content__app__image_container {
+	display: flex;
+}
+
+.banner__content__brand-mobile {
+	color: var(--neutral-08, #54555f);
+	font-size: 16px;
+	font-style: normal;
+	font-weight: 400;
+	line-height: 24px;
 }
 
-.banner__title {
-	font-size: 1.5rem;
-	letter-spacing: 0;
-	line-height: 1.75rem;
+.share__icon {
+	display: none;
+	height: 32px;
+	width: 32px;
+}
+
+.liferay__icon {
+	display: none;
+	height: 32px;
+	width: 32px;
+}
+
+.banner__content-title {
+	color: var(--neutral-10, #282934);
+	font-size: 23px;
+	font-style: normal;
+	font-weight: 600;
+	margin-left: 8px;
+	white-space: normal;
+}
+
+.banner__tag-container {
+	display: none;
+}
+
+.banner__tag-container-mobile {
+	margin: 16px 0;
 }
 
 .banner__product-tag {
 	background-color: #e6ebf5;
 	border-radius: 0.25rem;
 	color: #1c3667;
 	font-size: 0.8125rem;
-	margin: 0 0.625rem 0 0.625rem;
 	padding: 0.25rem 0.5rem 0.25rem 0.5rem;
+	white-space: nowrap;
```

Comment:
Can we use bootstrap to set padding nstead of using classes? And you can also reduce to use py-1 px-2, for example